analyzer-comments/python/two-fer/wrong_def_arg.md

```diff
@@ -1 +1 @@
-A value other than 'you' is used as a default argument.
+A value other than 'you' is bing used as a default argument.
```

Comment:
I would personally use DuckDuckGo 😉 😉 😉 (I mean: typo?)

Reply:
🤣 🤣 🤣   I am constitutionally unable to submit anything without at least **one** typo.  Many thanks for the catch.  😄